testsuite/tests/linear/should_compile/Linear3.hs

```diff
@@ -20,6 +20,6 @@ correctApp4 x = f (f x)
     f :: Int ⊸ Int
     f y = y
 
-correctIf :: Bool -> a ⊸ a
+correctIf :: Bool ⊸ a ⊸ a
```

Comment:
The multiplicity of the Boolean really doesn't matter. So either way.

Reply:
Well, this way we're checking more. In Linear4 this matters - it works with -> but not with ⊸. I've changed to ⊸ so that we can track it, should I revert?

Reply:
No, no. It's fine :slightly_smiling_face: